frontend/appflowy_flutter/lib/mobile/presentation/home/mobile_home_trash_page.dart

```diff
@@ -81,6 +71,71 @@ class MobileHomeTrashPage extends StatelessWidget {
   }
 }
 
+class _TrashActionAllButton extends StatelessWidget {
+  /// Switch between 'delete all' and 'restore all' feature
+  const _TrashActionAllButton({
+    this.isDeleteAll = false,
```

Comment:
Replace the boolean value with an enum here, as this will allow us to easily extend the available actions in the future.

```dart
enum { 
  deleteAll, 
  restoreAll;

  String get text {} 
  String get icon {}

}  
```

Reply:
I changed to the enum `_TrashActionType`
```
enum _TrashActionType {
  restoreAll,
  deleteAll,
}
```